src/components/ui/ShareMenu.jsx

```diff
@@ -0,0 +1,115 @@
+/* global _ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Telemetry from '../../libs/telemetry';
+
+const facebookShareUrl = location => {
+  return `https://www.facebook.com/sharer/sharer.php?u=${encodeURIComponent(location)}`;
+};
+
+const twitterShareUrl = location => {
+  return `https://twitter.com/intent/tweet?url=${ encodeURIComponent(location) }`;
+};
+
+const menu_height = 3 * 32;
+
+export default class ShareMenu extends React.Component {
+  _isMounted = false;
```

Comment:
You can get rid of this class property, you can already use this.state.open to know if the menu has to be close

Reply:
this way we'll not even need to use component lifecycle methods

Reply:
I didn't write this behavior, took it from the previous ContextMenu component.
Are you sure we can do without it? For example, when the poi panel is unloaded, shouldn't we tell the currently open ShareMenu to close itself?

Reply:
I'm not sure to understand your point. The behavior should be the same if you replace this check : https://github.com/QwantResearch/erdapfel/pull/663/files#diff-9f0fe226d7a5706cacfdfed978e5c974R56 with 
```js
if (this.state.open) {
  this.setState({ open: false });
}
```

Can you confirm ?

Reply:
yep, works great, thanks